dcs/task.py

```diff
@@ -1077,6 +1084,38 @@ def __eq__(self, other) -> bool:
         return False
 
 
+class CarpetBombing(Task):
+    """Create Carpet Bombing Task object
+
+    :param alt_above: AGL altitude (in meters) from which bombing is to be performed. Defaults to 2000.
+    :param pattern_length: The pattern length (in meters) of the carpet bombing run. Defaults to 500.
+    :param release_qty: How many weapons to release. Defaults to Expend.Auto. See :py:class:`dcs.task.Expend`
+    :param weapon_type: The weapon to be used. Defaults to WeaponType.Auto. See :py:class:`dcs.task.WeaponType`
+    """
+    Id = "CarpetBombing"
+    DEFAULT_ALT: int = 2000
+
+    def __init__(self, alt_above: int = DEFAULT_ALT,
+                 pattern_length: int = 500,
+                 release_qty: Expend = Expend.Auto,
+                 weapon_type: WeaponType = WeaponType.Auto) -> None:
+        super().__init__(self.Id)
+        # hardcoded parameters are present in .miz file, but not visible in Mission Editor
+        self.params = {
+            "altitude": alt_above,
+            "altitudeEnabled": True if alt_above != CarpetBombing.DEFAULT_ALT else False,
```

Comment:
I don't think this is actually right. I suspect this field is supposed to be "did the caller make a choice, or did they use the default?". This can't distinguish between a chosen altitude of 2000 and a default behavior. For that you need `alt_above: int | None = None`.

I've also got no idea what the impact of getting this wrong is. Might be just UI behavior in the ME? I won't block on this for now, but for future reference, that's how this thing ought to be written.

Reply:
Also, fyi, this is just a more verbose way to write `"altitudeEnabled": alt_above != CarpetBombing.DEFAULT_ALT`